plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -622,6 +624,31 @@ class NetworkingFormSection extends mixin(StoreMixin) {
       </span>
     );
 
+    if (networks != null && networks.length > 1) {
+      networks = networks.map(function({ mode, name }) {
+        return {
+          name,
+          mode: Networking.internalToJson[mode]
+        };
+      });
+
+      return (
+        <div>
+          <h2 className="flush-top short-bottom">
+            Networking
+          </h2>
+          <p>
+            {"This service has advanced networking configuration, which we" +
```

Comment:
as discussed off line: there's an issue with our setup so we couldn't just add i18n here, but since this is a quick fix to be removed in the following sprints we decided not to waste any more time here.